Firmware failes to compile, littlefs bug

**Describe the bug**
Firmware fails to compile due to littlefs library error.

**To Reproduce**
Steps to reproduce the behavior:
1. Compile firmware

**Expected behavior**
Build firmware without error.

**Screenshots**
```
Compiling .pio/build/esp32/libba2/LittleFS_esp32/lfs.c.o
.pio/libdeps/esp32/LittleFS_esp32/src/esp_littlefs.c: In function 'esp_vfs_littlefs_register':
.pio/libdeps/esp32/LittleFS_esp32/src/esp_littlefs.c:208:9: error: unknown field 'utime_p' specified in initializer
         .utime_p     = &vfs_littlefs_utime,
         ^
.pio/libdeps/esp32/LittleFS_esp32/src/esp_littlefs.c:208:24: warning: initialization from incompatible pointer type [-Wincompatible-pointer-types]
         .utime_p     = &vfs_littlefs_utime,
                        ^
.pio/libdeps/esp32/LittleFS_esp32/src/esp_littlefs.c:208:24: note: (near initialization for 'vfs.<anonymous>.access_p')
.pio/libdeps/esp32/LittleFS_esp32/src/esp_littlefs.c:179:27: warning: missing braces around initializer [-Wmissing-braces]
     const esp_vfs_t vfs = {
                           ^
.pio/libdeps/esp32/LittleFS_esp32/src/esp_littlefs.c:179:27: note: (near initialization for 'vfs')
*** [.pio/build/esp32/libba2/LittleFS_esp32/esp_littlefs.c.o] Error 1
```

--- a/firmware/platformio.ini
+++ b/firmware/platformio.ini
@@ -15,7 +15,7 @@ framework = arduino
 lib_deps = 
 	hieromon/AutoConnect@^1.1.7
 	mathertel/RotaryEncoder@^1.3.0
-	lorol/LittleFS_esp32@^1.0
+	lorol/LittleFS_esp32@1.0.0
 monitor_speed = 115200
 build_flags = 
 	-D CORE_DEBUG_LEVEL=5

